lib/internal/bootstrap/node.js

```diff
@@ -155,6 +155,19 @@
     });
     process.argv[0] = process.execPath;
 
+    // Handle inspector security warning
+    const debugOptions = process.binding('config').debugOptions;
+    if (debugOptions.host !== '127.0.0.1') {
```

Comment:
What about other loopback addresses? What about IPv6?

Reply:
This is just a guard against default value that is here, even when the user does not run node with inspect parameter